transform/run_pipelines.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+set -o nounset
+# Uncomment this line to see each command for debugging (careful: this will show secrets!)
+# set -o xtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+FACTORY_NAME=$(cd "${SCRIPT_DIR}"/../infrastructure/transform && terragrunt output adf_name | tr -d '"')
+RESOURCE_GROUP_NAME=$(cd "${SCRIPT_DIR}"/../infrastructure/core && terragrunt output core_rg_name | tr -d '"')
```

Comment:
Should we validate that we get something back from these two calls?
i.e. exit early if FACTORY_NAME and RESOURCE_GROUP_NAME aren't set?

If so, I wonder if this should be wrapped in a function. Do we do this elsewhere?

Reply:
We have `set -o nounset` at the top of the script so this shouldn't be a problem?

Reply:
OK. Does that catch empty strings too? If so, all good.

Reply:
Added catching empty strings